PKGBUILD

```diff
@@ -35,7 +35,7 @@ build() {
     yarn --non-interactive --frozen-lockfile
     NODE_ENV=production yarn --non-interactive --frozen-lockfile build
   )
-  go run github.com/gobuffalo/packr/packr
+  go run hack/packr/packr.go
   local build_date=$(date "+%F-%T" -d "@${SOURCE_DATE_EPOCH}")
```

Comment:
force utc time, remove quotes around format and move it to back:

`date -u -d "@${SOURCE_DATE_EPOCH}" +%F-%T`